operator/pkg/controller/istiocontrolplane/istiocontrolplane_controller.go

```diff
@@ -329,36 +336,43 @@ func mergeIOPSWithProfile(iop *iopv1alpha1.IstioOperator) (*v1alpha1.IstioOperat
 	if hub != "" && hub != "unknown" && tag != "" && tag != "unknown" {
 		buildHubTagOverlayYAML, err := helm.GenerateHubTagOverlay(hub, tag)
 		if err != nil {
+			metrics.CountCRMergeFail(metrics.CannotMergeProfileWithHubTagError)
 			return nil, err
 		}
 		profileYAML, err = util.OverlayYAML(profileYAML, buildHubTagOverlayYAML)
 		if err != nil {
+			metrics.CountCRMergeFail(metrics.CannotMergeProfileWithHubTagError)
```

Comment:
this error is probably too specific as a metric label. OverlayError maybe? here and above

Reply:
Fixed it.